app/routes/labs._index.mdx

```diff
@@ -0,0 +1,293 @@
+
+import React from 'react'
+
+import styles from './labs/across.css'
+import mma from './labs/mma.png'
+
+  <div style={{ backgroundColor: '#0c0f18', color: 'white', minHeight: '2000px' }}>
+
+    <ParallaxImage
+      src="https://svs.gsfc.nasa.gov/vis/a010000/a012700/a012740/NeutronStarMerger_Shot1_4KStill4_v01_print.jpg"
+      alt="BNS"
+    />
+
+    <GridContainer className="usa-section" style={{minWidth:'75%'}}>
+
+      <div className="section-head"><h1>Overview</h1></div>
+
+      <div className="container" style={{ marginTop: '0px', display: 'flex', justifyContent: 'space-between' }}>
+        <div className="usa-paragraph text-base-lightest magin-top-0" style={{
+            fontSize: '1.1rem',
+            zIndex: 1,
+            whiteSpace: 'normal',
+            width: '90%',
+            marginTop: '0px'
+
+          }}>
+
+          The emerging field of multimessenger astronomy combines the observation of disparate messenger signals to reveal unique
+          information about astrophysical sources. The four extrasolar messengers that make up multimessenger astronomy are electromagnetic
+          radiation, gravitational waves, neutrinos, and cosmic rays and the primary multimessenger sources outside the heliosphere are
+          expected to be compact binary pairs (black holes and neutron stars), supernovae, irregular neutron stars, gamma-ray bursts,
+          active galactic nuclei, and relativistic jets. The present study of the these multimessenger sources has become one of the most
+          promising discovery areas of modern astronomy. Identified as one of the central themes of the 2020 Decadal Review, the full
+          realization of time-domain and multi-messenger astronomy now stands to open an entirely new window on the Universe.
+
+          The first joint detection of two merging neutron stars in gravitational waves and electromagnetic radiation on August 17, 2017
+          was a watershed moment for the field of multimessenger astronomy. The information gained from the detection of both messengers
+          enable science that would have been impossible to perform with either signal alone. The discovery demonstrated that multimessenger
+          astrophysics offers a powerful methodology of growing importance, as astronomers combine electromagnetic radiation, gravitational
+          wave radiation, and particle astrophysics observations of cosmic events.
+
+
+          Multimessenger.nasa.gov aims to support this emerging field by consolidating important information on resources available to
+          scientists around the world to enable future discoveries.
+
+        </div>
+
+        <img
+          src={mma}
+          alt="Descriptive Alt Text"
+          style={{
+            width: 'auto',
+            height: '600px',
+            marginLeft: '0px'
+          }}
+        />
+      </div>
+
+      <div className="section-head" style={{marginTop:'150px'}}></div>
```

Comment:
Why is there an empty div?

Reply:
I decided to have a section header with an optional section title that would separate each major segment of the page. The contents of each segment could easily be moved into the section instead.

Reply:
I've gone ahead and created the same effect by using Grid rows rather than custom sections.